### DISCUSSION TITLE
Cant connect with Unifi-network Android app anymore

### DISCUSSION THREAD

**Discussion by TheQue42** [goofball222/unifi#90] [Q&A] [answered]
I upgraded to 6.2.26.0 a few days ago or something.

I just noticed that I can no longer connect with the Unifi Android app? Anyone no if this is solvable somehow, or is it unifi that have messed up something?

**Comment by goofball222** [comment]
I tried from Android to my 6.3 beta controller and had no problems. I'm also fairly certain I was able to connect and manage it when it was 6.2.25/26 as well. Make sure you have everything listed as "Ingress Ports required for L3 management over the internet" on https://help.ui.com/hc/en-us/articles/218506997-UniFi-Ports-Used mapped to the controller.

**Reply by TheQue42** [reply]
I'll do some additional sniffing, but I havent changed anything (intentionally at least) in my setup, I just noticed it stopped working all of a sudden, and I use the mobile app quite often.
So unless portainer messed up in the container upgrade, it should all be good. Do you happen to know which ports are used for the authentication from the APP? Or does it use the :8443 just like the web browser?

**Comment by TheQue42** [comment, marked as answer]
Ok, after a complete removal of the docker stack (mongo+unifi) and a restore of a settings-only DB, it seems I am back up again. Sorry for the FUD.
